pkgs/by-name/as/asciiquarium-transparent/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  perlPackages,
+  ...
+}:
+stdenv.mkDerivation {
+  pname = "asciiquarium-transparent";
+  version = "unstable-2023-02-19";
+  src = fetchFromGitHub {
+    owner = "nothub";
+    repo = "asciiquarium";
+    rev = "653cd99a611080c776d18fc7991ae5dd924c72ce";
+    hash = "sha256-72LRFydbObFDXJllmlRjr5O8qjDqtlp3JunE3kwb5aU=";
+  };
+  nativeBuildInputs = [makeWrapper];
+  buildInputs = [perlPackages.perl];
+  installPhase = ''
+    mkdir -p $out/bin
+    cp asciiquarium $out/bin/asciiquarium
+    wrapProgram $out/bin/asciiquarium --set PERL5LIB ${perlPackages.makeFullPerlPath [perlPackages.TermAnimation]}
+  '';
+  meta = with lib; {
+    description = mdDoc ''
```

Comment:
```suggestion
    description = ''
```

Reply:
why not mdDoc? aren't we supposed to migrate away from asciidoc?

Reply:
What you say is true for descriptions in modules, but `meta.description` in packages is in plain text, as far as I know.

Reply:
`mdDoc` is also a no-op, actually:

https://github.com/NixOS/nixpkgs/blob/558bcddd7b83ceb962cb4405a92f4060cb98cb3e/lib/options.nix#L346-L349